11/centos/centos7/hotspot/Dockerfile

```
@@ -1,4 +1,5 @@
-FROM eclipse-temurin:11.0.20.1_1-jdk-centos7 as jre-build
+ARG JAVA_VERSION=11.0.20.1_1
+FROM eclipse-temurin:"$JAVA_VERSION"-jdk-centos7 as jre-build
```

```suggestion
FROM eclipse-temurin:"${JAVA_VERSION}"-jdk-centos7 as jre-build
```

To stay coherent with the Dockerfile already using an `ARG` (and it is a good practise)